eks-anywhere-common/Addons/Partner/Sysdig/sysdig.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: sysdig
+  namespace: sysdig-agent
```

Comment:
This is incorrect. ns is `sysdig`. Is your software tested before PR?

Reply:
Namespace has been relocated as requested